### PR TITLE
[Import Group] Stat Var related API clean up

### PR DESCRIPTION
**Pull request by shifucun**
In general, there is no need to merge for stat var related API, while the preferred cache table should be used. This PR

- Clean up logic throughout stat var APIs by using bigtable.Read() function
- Remove the condition to build stat var hierarchy using triples. That's only used during hierarch dev stage.
- Split functions into separate folders for readability.
- Restrict some function parameter scope from "store" to "bigtable group".

### CODE REVIEW

File: internal/server/statvar/statvar_group.go [hunk 1]
```diff
@@ -169,61 +173,18 @@ func GetStatVarGroupNode(
 	}
 
 	result := &pb.StatVarGroupNode{}
-
-	if in.GetReadFromTriples() {
```
Comment (pradh) [inline internal/server/statvar/statvar_group.go:173]: is GetReadFromTriples being deprecated?

**Review by pradh — APPROVED** [review]
Nice cleanup!  (though might be easier to review when function moves across files are in their own logic-less PR)

File: internal/server/statvar/statvar_group.go (hunk 1 above)
Comment (shifucun) [reply internal/server/statvar/statvar_group.go:173]: Yep, this is not used anywhere now.

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**